### ISSUE TITLE
.irssi/config encoding settings are not reflected in macirssi nicklist.

### ISSUE THREAD

**Issue by LoptimistK** [MacIrssi/MacIrssi#36]
recode = "yes";
recode_fallback = "cp949";
recode_out_default_charset = "cp949";
recode_transliterate = "yes";

these sets are reflected for chats but nicklist still uses different encoding, thus displayed in ascii/utf-8 not cp949 for stable release (8.6.6), and even worse, 8.6.7 pre-releases show blank displays for nicknames that are non-ascii.

**Comment by ghost** [comment]
So the latest betas attempt to force Irssi to use UTF-8 (and assume that all content from Irssi is in UTF-8). Character sets should be extinct by now in MacIrssi.

If you remove the recode character sets from your config, do you get the output in MacIrssi you would expect?

**Comment by LoptimistK** [comment]
nope. using utf-8 won't work for me because my locale is english/us.

what I see is something along this line @Admin µð¾¾ÀÎ»çÀÌµå¸¦ µé¶ô³¯¶ôÇØ¼­
which should look like: @Admin ㅇㅇ

**Comment by ghost** [comment]
UTF-8 _should_ work if all the locales and character sets are set to UTF-8. It's likely that I've missed some that I need to reset in MacIrssi.

**Comment by LoptimistK** [comment]
http://forums.gentoo.org/viewtopic-t-346190-start-0.html

quick google search shows this, and i think this is correct.

i need to have my locale as ko_kr in order for utf-8 should work.

**Comment by ghost** [comment]
So I'm interested in trying to make sure this stuff works. What I could do with from you is a set of steps to reproduce problems you're seeing. Ideally, reproducible in an empty chat room. Including:
1. MacIrssi and irssi config settings, for both broken states and working (if you have them)
2. Steps to reproduce, including what text you entered and in what mac keyboard layout.
3. Your expectation of what should happen.

My aim is for char sets to work with as little set up as possible. It should not normally be needed for you to set any locales, charsets, etc. I'd like to make that the case if it is not currently.
